components/notes/CreateProspectNote.tsx

```diff
@@ -0,0 +1,95 @@
+import { useState } from 'react'
+import { Box, Button, TextField } from '@mui/material'
+import CancelIcon from '@mui/icons-material/Cancel'
+import CreateIcon from '@mui/icons-material/Create'
+import SaveIcon from '@mui/icons-material/Save'
+
+function CreateProspectNote({ prospect }) {
```

Comment:
let's name this just NoteTextarea (in the future we'll use this as the edit component as well). Also let's rename variables that mention the prospect. We might want in the future to use notes for other things as well